src/cli/domain/get-components-deps.js

```diff
@@ -17,7 +19,7 @@ module.exports = function(components){
   components.forEach((c) => {
     const pkg = fs.readJsonSync(path.join(c, 'package.json'));
     const type = pkg.oc.files.template.type;
-    const dependencies = pkg.dependencies;
+    const dependencies = pkg.dependencies || {};
```

Comment:
this comes from package.json. After testing, I found a component with no dependencies props could actually generate some troubles